pkgs/os-specific/linux/joycond-cemuhook/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, python3Packages, fetchFromGitHub}:
+
+python3Packages.buildPythonPackage {
```

Comment:
```suggestion
python3Packages.buildPythonApplication {
```

Reply:
What is the difference between these two? I assume `...buildPythonPackage` is used more for library or library/application combos than just applications?

Reply:
The differences are listed in the [docs](https://ryantm.github.io/nixpkgs/languages-frameworks/python/#buildpythonapplication-function). Generally `buildPythonPackage` is used when the project is supposed to be available as a python library.